source/common/common/to_lower_table.cc

```diff
@@ -0,0 +1,16 @@
+#include "common/common/to_lower_table.h"
+
+ToLowerTable::ToLowerTable() {
```

Comment:
Ah I see. Makes sense. That's a good tip, not sure if it's worth it in this case though so I think I will skip.